src/Jobs/BasePackageJob.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Flagrow\Bazaar\Jobs;
+
+use Carbon\Carbon;
+use Flagrow\Bazaar\Composer\ComposerCommand;
+use Flagrow\Bazaar\Composer\ComposerEnvironment;
+use Flagrow\Bazaar\Composer\ComposerOutput;
+use Flagrow\Bazaar\Task;
+//use Illuminate\Bus\Queueable;
+//use Illuminate\Queue\SerializesModels;
+//use Illuminate\Queue\InteractsWithQueue;
+//use Illuminate\Contracts\Queue\ShouldQueue;
```

Comment:
The `queue` package of Laravel is really easy to use outside of Laravel. We could register our own queue driver (sync is the most basic one and it will work anywhere) and run the jobs through it. When Flarum implements queues we can simply switch the dispatch command to push the job to Flarum queue driver. 

At the moment I simply created a method that runs the `handle` method of the job. 

If no work has been made on queues for Flarum, maybe we should create a `flagrow/queue` package that register a queue worker and allow users to choose a driver. Then require this extension in all our extensions requiring queues.